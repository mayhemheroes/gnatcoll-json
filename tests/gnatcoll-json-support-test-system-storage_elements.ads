with System.Storage_Elements; use System.Storage_Elements;
with AUnit.Test_Cases;
package GNATCOLL.JSON.Support.Test.System.Storage_Elements is
   type Test_Case is new AUnit.Test_Cases.Test_Case with record
      T_0 : Storage_Offset;
      T_1 : Storage_Element;
      T_2 : Storage_Array (1 .. 10);
      T_3 : Integer_Address;
   end record;

   overriding procedure Set_Up_Case (Test : in out Test_Case);
   overriding procedure Register_Tests (Test : in out Test_Case);
   overriding function Name (Test : Test_Case) return AUnit.Message_String;

end GNATCOLL.JSON.Support.Test.System.Storage_Elements;
