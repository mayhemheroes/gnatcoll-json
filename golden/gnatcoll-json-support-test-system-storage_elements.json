{
  "T_0": 305442814,
  "T_1": 135,
  "T_2": [
    1,
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10
  ],
  "T_3": 3402287818
}
